flow-typed/npm/downshift_vx.x.x.js.flow

```diff
@@ -0,0 +1,180 @@
+/**
```

Comment:
I don't know very much about how this file is generated, but I would expect the file to have a more specific version: `flow-typed/npm/downshift_v1.x.x.js.flow`

Reply:
this PR is against next branch, so it would be `downshift_v2.x.x.js.flow`